proto/ibc/apps/cross-chain-validation/tx.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package ibc.apps.ccv.v1;
+
+option go_package = "github.com/cosmos/ibc-go/modules/apps/ccv/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// MsgSubmitNewChain defines a msg to submit a new baby chain that can be validated
+// by the parent chain validators.
+message MsgSubmitNewChain {
+  option (gogoproto.goproto_getters) = false;
+  
+  // the proposed chain-id of new chain
+  string chain_id = 1 [(gogoproto.moretags) = "yaml:\"chain_id\""];
+  // minimum stake that must commit to validate new chain before it can start
+  cosmos.base.v1beta1.Coin minimum_stake = 2 [(gogoproto.nullable) = false];
+  // optional grace period to allow validators to join initial validator set after minimum stake is reached
+  // grace period is a duration in nanoseconds
+  uint64 grace_period = 3 [(gogoproto.moretags) = "yaml:\"grace_period\""];
```

Comment:
Time-based units in a module can lead to problems during upgrades or chain halts. In general, I suggest sticking to block-based units.

In any case, I suggest removing `grace_period` since it doesn't seem like prerequisite to core ccv functionality.